cpp/ql/test/library-tests/ir/ssa/aliased_ssa_ir.expected

```diff
@@ -1502,3 +1502,90 @@ ssa.cpp:
 #  310|     v310_12(void)                = ReturnVoid                   : 
 #  310|     v310_13(void)                = AliasedUse                   : m310_3
 #  310|     v310_14(void)                = ExitFunction                 : 
+
+#  319| void DoubleIndirectionEscapes(char*)
+#  319|   Block 0
```

Comment:
I can't review this test. Please rebase the PR so the test comes first in a separate commit, followed by a commit that makes the QL changes and shows how the test output changes.